report/report.go

```diff
@@ -0,0 +1,12 @@
+package report
+
+import ()
+
+type ReportableElement interface {
+	GetHeaders() []string
```

Comment:
These are the tabular headers (i.e. map keys) correct? Would "keys" be more meaningful?